src/webresource-handler/index.ts

```diff
@@ -81,22 +81,30 @@ const isFileInValidPath = async (
 		url: req.url,
 		method: req.method,
 	});
-	const resourceName = sbvrUtils.resolveSynonym(odataRequest);
+	const sqlResourceName = sbvrUtils.resolveSynonym(odataRequest);
+
+	const table = model.abstractSql.tables[sqlResourceName];
+
+	if (table == null) {
+		return false;
+	}
 
 	const permission = req.method === 'POST' ? 'create' : 'update';
 	const vocab = model.versions[model.versions.length - 1];
 
 	// Checks if it has permissions on both the original resourceName or any synonym
-	const hasPermissions =
-		(await checkPermissions(req, permission, resourceName, vocab)) ||
```

Comment:
I removed the permission check on `resourceName` since it's a sql name and as such won't (or at least shouldn't) have permissions associated